gotham/src/handler/assets/mod.rs

```diff
@@ -251,6 +275,52 @@ fn create_file_response(options: FileOptions, state: State) -> Pin<Box<HandlerFu
         .boxed()
 }
 
+// Checks for existence of "Range" header and whether it is in supported format
+// This implementations only supports single part ranges.
+// Returns a result of length and optional starting position, or an error if range value is invalid
+// If range header does not exist or is unsupported the length is the whole file length and start position is none.
```

Comment:
```suggestion
/// Checks for existence of "Range" header and whether it is in supported format
/// This implementations only supports single part ranges.
/// Returns a result of length and optional starting position, or an error if range value is invalid
/// If range header does not exist or is unsupported the length is the whole file length and start position is none.
```

Please use doc comments to document functions.